packages/contracts-bedrock/test/DeployAuthSystem.t.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { Test } from "forge-std/Test.sol";
+import { stdToml } from "forge-std/StdToml.sol";
+import { Solarray } from "scripts/libraries/Solarray.sol";
+
+import { DeployAuthSystemInput } from "scripts/DeployAuthSystem.s.sol";
+
+contract DeployAuthSystemInput_Test is Test {
+    DeployAuthSystemInput dsi;
```

Comment:
`dsi` was shorthand for Deploy Superchain Input. Feel free to use `dasi` if you like that better
```suggestion
    DeployAuthSystemInput dai;
```

Reply:
Ah yeah. I started this before #11845, so thought maybe I should go with that, but agree `dasi` is nicer.